### PR TITLE
[13.0][OU-FIX] stock_picking_send_by_mail: Migration script.

### PR DESCRIPTION
**Pull request by victoralmau**
Since migration https://github.com/OCA/stock-logistics-workflow/pull/789 some data needs to be updated to avoid `format_tz()` error when using mail template. 

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT33312

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Although this can be correct, shouldn't we switch to standard Odoo system?

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by HaraldPanten** [comment]
Shouldn't you remove "TT33312" from the commit message?

**Comment by pedrobaeza** [comment]
No, that gives us the trace. Odoo does the same.

**Review by HaraldPanten — APPROVED** [review]

File: stock_picking_send_by_mail/migrations/13.0.1.2.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr,
+        "stock_picking_send_by_mail",
+        "migrations/13.0.1.2.0/noupdate_changes.xml",
+    )
```
Comment (pedrobaeza) [inline stock_picking_send_by_mail/migrations/13.0.1.2.0/post-migration.py:13]: You should delete translations as well.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
I see that you have changed the source term in the translations. Are that translations completed? If so, you should change the translation as well with the correct expression.

**Comment by victoralmau** [comment]
Previously, the term origin was not updated correctly, now the term origin is updated, and in the different translations the corresponding expression is added.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**